opmode/auton/AFR.java

```diff
@@ -0,0 +1,55 @@
+package org.firstinspires.ftc.teamcode.opmode.auton;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.util.ElapsedTime;
+
+import org.firstinspires.ftc.teamcode.hardware.Camera;
+import org.firstinspires.ftc.teamcode.hardware.Claw;
+import org.firstinspires.ftc.teamcode.hardware.Drivetrain;
+import org.firstinspires.ftc.teamcode.hardware.Hook;
+import org.firstinspires.ftc.teamcode.hardware.Lift;
+
+@Autonomous(name="AFR")
```

Comment:
this is a terrible name for a class. Better naming convention, use super classes or interfaces so that there is a base op mode for init and run opmode is diff or smth